addons/binding/org.openhab.binding.kodi/src/main/java/org/openhab/binding/kodi/internal/model/KodiPVRChannelGroup.java

```diff
@@ -0,0 +1,42 @@
+/**
+ * Copyright (c) 2010-2018 by the respective copyright holders.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.binding.kodi.internal.model;
+
+/**
+ * Class representing a Kodi PVR channel group
+ *
+ * @author Christoph Weitkamp - Initial contribution
+ */
+public class KodiPVRChannelGroup extends KodiBaseItem {
+    /**
+     * The PVR channel group id
+     */
+    private int channelGroupId;
+
+    /**
+     * The PVR channel type
+     */
+    private String channelType;
+
+    public int getId() {
+        return channelGroupId;
+    }
+
+    public void setId(final int channelGroupId) {
```

Comment:
Also `channelGroupId` is referred to as `channelGroupID` elsewhere in the code (e.g. the method `getChannelGroupID`).

Reply:
See above.